### DISCUSSION TITLE
How do I backup my existing Gitlens settings and restore to a new VS Code installation with virgin Gitlens extension installed?

### DISCUSSION THREAD

**Discussion by mkanet** [gitkraken/vscode-gitlens#3446] [Q&A]
Lets say I already have source control setup on a clean installation of VSCode.  Could someone tell me an easy way to backup my existing Gitlens settings and restore them on another installation of VSCode that I just installed Gitlens on (assuming I already have basic source control setup already on the new PC)?  I only need to backup the settings for this extension only.

**Comment by GabenGar** [comment]
Do you have keys starting with `gitlens.*` in your user settings file?

**Comment by d13** [comment]
There's a settings sync feature in VS Code: https://code.visualstudio.com/docs/editor/settings-sync
